src/Debugger/Impl/rtvs/R/overrides.R

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See LICENSE in the project root for license information.
+
+view <- function(x, title) {
+  if(is.function(x) || is.data.frame(x) || is.table(x) || 
+     is.matrix(x) || is.list(x) || is.ts(x) || length(x) > 1) {
+    if (missing(title)) {
+      title <- ""
+    }
+    invisible(rtvs:::send_message('View', deparse(substitute(x)), title))
+  } else {
+    print(x)
+  }
+}
+
+browser <- function(url) {
```

Comment:
We really should rename it, because now it's clashing with `base::browser` (only inside this package, but still). Let's call it `open_url` or something like that, and rename the message as well.